cucumberOpts incompatible type

Hi there,
I'm using protractor 4.0.13, typescript 2.1.4 and cucumber 1.3.1.
In my config.ts I've got such config in cucumberOpts:

`    cucumberOpts: {
        require: [
            '../steps/**/*Steps.js',
            '../support/hooks.js'
        ],
        format: [
            'pretty',
            'json:report.json'
        ]
    },`


Every time when I run tests I got such error:
`config/config.ts(3,12): error TS2322: Type '{ seleniumAddress: string; baseUrl: string; capabilities: { browserName: string; chromeOptions: {...' is not assignable to type 'Config'.
  Types of property 'cucumberOpts' are incompatible.
    Type '{ require: string[]; format: string[]; }' is not assignable to type '{ [key: string]: any; require?: string; tags?: string; format?: string; coffee?: any; noSnippets?...'.
      Types of property 'require' are incompatible.
        Type 'string[]' is not assignable to type 'string'.
`

The clue is in config.ts file. In lib/config.ts:563 and lib/config.ts:572 you've got 
`    require?: string;`
Could you please change them into:
`    require?: Array<string>;`

Regards,
Kacper

--- a/docs/frameworks.md
+++ b/docs/frameworks.md
@@ -82,19 +82,19 @@ exports.config = {
   seleniumAddress: 'http://localhost:4444/wd/hub',
 
   baseUrl: 'https://angularjs.org/',
-  
+
   capabilities: {
       browserName:'chrome'
   },
 
-  framework: 'custom',  // set to "custom" instead of cucumber. 
-  
+  framework: 'custom',  // set to "custom" instead of cucumber.
+
   frameworkPath: require.resolve('protractor-cucumber-framework'),  // path relative to the current config file
-  
+
   specs: [
     './cucumber/*.feature'     // Specs here are the cucumber feature files
   ],
-  
+
   // cucumber command line options
   cucumberOpts: {
     require: ['./cucumber/*.js'],  // require step definition files before executing features
@@ -115,5 +115,3 @@ Using a Custom Framework
 ------------------------
 
 Check section [Framework Adapters for Protractor](/lib/frameworks/README.md) specifically [Custom Frameworks](/lib/frameworks/README.md#custom-frameworks)
-
-

--- a/lib/config.ts
+++ b/lib/config.ts
@@ -1,6 +1,8 @@
 import {PluginConfig} from './plugins';
 
 export interface Config {
+  [key: string]: any;
+
   // ---------------------------------------------------------------------------
   // ----- How to connect to Browser Drivers -----------------------------------
   // ---------------------------------------------------------------------------
@@ -559,30 +561,6 @@ export interface Config {
    * See the full list at http://mochajs.org/
    */
   mochaOpts?: {[key: string]: any; ui?: string; reporter?: string;};
-
-  /**
-   * Options to be passed to Cucumber (when set up as a custom framework).
-   */
-  cucumberOpts?: {
-    [key: string]: any;
-    /**
-     * Require files before executing the features.
-     */
-    require?: string[] | string;
-    /**
-     * Only execute the features or scenarios with tags matching @dev.
-     * This may be an array of strings to specify multiple tags to include.
-     */
-    tags?: string;
-    /**
-     * How to format features (default: progress)
-     */
-    format?: string[] | string;
-    /**
-     * Quickly scan your features without actually running them.
-     */
-    dryRun?: boolean;
-  };
 
   /**
    * See docs/plugins.md

